volatility3/framework/plugins/windows/registry/getcellroutine.py

```diff
@@ -0,0 +1,98 @@
+# This file is Copyright 2019 Volatility Foundation and licensed under the Volatility Software License 1.0
```

Comment:
Extremely unimportant but the 2019 feels like a simple copy and paste mix-up, is it worth changing to 2024?

Reply:
I will fix this. I started by making a copy of userassist.py since it had all the imports needed and call to hivelist

Reply:
I do exactly the same thing every time 😅